Makefile

```diff
@@ -52,40 +52,40 @@ ARCH=$(shell uname -m)
 build: $(MICROSERVICES)
 
 cmd/core-metadata/core-metadata:
-	$(GO) build $(GOFLAGS) -o $@ ./cmd/core-metadata
+	$(GOCGO) build $(GOFLAGS) -o $@ ./cmd/core-metadata
```

Comment:
Since go-mod-bootstrap now references go-mod-messaging, all the service need `GOCGO` due to the implicit ZMQ dependency.